extra/cockpit/foreman-cockpit-session

```diff
@@ -170,24 +170,32 @@ class Relay
   def initialize(proxy, params)
     @proxy = proxy
     @params = params
+    @inject_authorization = @params['ssh_user'] != 'root' && @params['effective_user_password']
```

Comment:
I'm new here, doesn't root still needs to provide a password?..

Reply:
This is only effective user password which gets used when attempting to elevate permissions. When the connection is established as a non-root user, this password can then used to attempt becoming root.